src/main/java/uk/nhs/tis/trainee/notifications/event/FormListener.java

```diff
@@ -72,7 +72,7 @@ public void handleFormUpdate(FormUpdateEvent event) throws MessagingException {
 
     String traineeId = event.traineeId();
     emailService.sendMessageToExistingUser(traineeId, FORM_UPDATED, templateVersion,
-        templateVariables);
+        templateVariables, null);
```

Comment:
In this and similar calls - the tisReference is left null for now, since its not directly relevant to the ticket. Populating this would be for a later slice I think.